src/models/device.cpp

```diff
@@ -44,6 +44,20 @@ QString Device::currentDeviceName() {
   return deviceName;
 }
 
+// static
+QString Device::currentDeviceReport() {
+  QString buffer = "";
+  buffer.append("\nDevice Report \n ============ \n");
+
+  buffer.append(QString("Name->%0 \n").arg(currentDeviceName()));
```

Comment:
1. remove the extra space after "%0"
2. should we use %1 instead of 0? See https://doc.qt.io/qt-5/qstring.html#arg